R/backend-dbplyr__duckdb_connection.R

```diff
@@ -77,6 +77,26 @@ duckdb_grepl <- function(pattern, x, ignore.case = FALSE, perl = FALSE, fixed =
   }
 }
 
+duckdb_n_distinct <- function(..., na.rm = FALSE) {
+  check_na_rm <- pkg_method("check_na_rm", "dbplyr")
+  glue_sql2 <- pkg_method("glue_sql2", "dbplyr")
```

Comment:
Those are private functions. @mgirlich: Is it fair to assume that these will stay around?

Reply:
`check_na_rm()` should definitely stay and might be worth exporting for backends. I opened an issue: https://github.com/tidyverse/dbplyr/issues/1483.
It would also make sense to export `glue_sql2()` but I don' think this is ready yet. So, I guess this is more likely to evolve (in particular I would consider changing the name before exporting it).